server_app/user/service.js

```diff
@@ -0,0 +1,56 @@
+const UserRepository = require("./model");
+const encriptor = require("../helpers");
+
+const login = async ({username, password, sessionId}) => {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const userDocument = await UserRepository.findOne({
+        "login": username,
+        "password": encriptor.Crypto(password, username),
+      });
+    
+      if(userDocument != null) {
+        return resolve({
+          "sessionId": sessionId,
```

Comment:
Miltão, What do you think about leaving the about leaving the responsibility of setting up the user session to the handler?